doc/source/extending.rst

```diff
@@ -79,10 +79,10 @@ on :ref:`ecosystem.extensions`.
 
 The interface consists of two classes.
 
-``ExtensionDtype``
-^^^^^^^^^^^^^^^^^^
+:class:`pandas.api.extension.ExtensionDtype`
```

Comment:
maybe add a `~` in the title link (to only show ExtensionDtype), or in the link in the text. (using the full link once is enough I think?)